org.eclipse.xtend.core.tests/xtend-gen/org/eclipse/xtend/core/tests/EmfModelsTest.java

```diff
@@ -225,48 +225,48 @@ private Object getDefaultValue(final Class<?> clazz) {
     boolean _isPrimitive = clazz.isPrimitive();
     if (_isPrimitive) {
       boolean _matched = false;
-      if (Objects.equal(clazz, Boolean.TYPE)) {
+      if (com.google.common.base.Objects.equal(clazz, Boolean.TYPE)) {
```

Comment:
Why here there's still Objects.equal? I've seen that also below in another file

Reply:
That remained since it is generated by the `XbaseCompiler` which is adapted just in this PR and my development-eclipse doesn't have this fix yet. I oversaw that since when I didn't have https://github.com/eclipse/xtext/pull/2992 in the same branch and also had my search configured to not contain derived resources.

I have now used a secondary Eclipse launched from my Xtext workspace to regenerated all the java files that still contained Guava's Objects class.
Now, with this change I don't find any occurrence of `com.google.common.base.Objects` in my Xtext workspace, not even in derived resources.

Reply:
Usually we do this update with a milestone update for the xtend version in the root pom

Reply:
@HannesWell I didn't think about that.

I typically use the latest nightly in my Xtext workspace.

@cdietrich do you prefer to keep the old version of the generated files for the moment?

Reply:
Do you have Cappa to regen all

Reply:
Also is this about fragment or about xtend 
If xtend regen with the m2 boot strap

Reply:
@cdietrich this is not about the fragment but about a change in XbaseCompiler, so the regenerated Java code from Xtend could be done once we install the new Xtend in the development workspace (i.e., when this is merged to master, the nightly p2 site is published, and the development workspace takes Xtend from the nightly p2 site). When we publish M1, the xtend maven plugin will also behave accordingly.

Reply:
Yes this is why I hesitate to regen rn. If we can burn milestones we could build it now after merge and then for the real milestone but I guess it’s not Wort or